src/XMLimport.cpp

```diff
@@ -114,9 +114,8 @@ bool XMLimport::importPackage( QIODevice * device, QString packName, int moduleF
         mpAlias->setPackageName( mPackageName );
         mpAlias->setName( mPackageName );
         mpAlias->setIsFolder( true );
-        QString _nothing = "";
-        mpAlias->setScript(_nothing);
-        mpAlias->setRegexCode("");
+        mpAlias->setScript( QString() );
+        mpAlias->setRegexCode( QString() );
```

Comment:
Cleaner code that won't trip things when we start looking for string literals as part of __I18n__ work.